### PR TITLE
import finished trial data to tuner when experiment is resumed

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: tools/nni_cmd/nnictl_utils.py [hunk 1]
```diff
@@ -482,25 +461,25 @@ def export_trials_data(args):
...
             if args.type == 'json':
-                json_records = []
-                for trial in records:
-                    value = trial.pop('reward', None)
-                    trial_id =  trial.pop('id', None)
-                    json_records.append({'parameter': trial, 'value': value, 'id': trial_id})
-            with open(args.path, 'w') as file:
-                if args.type == 'csv':
-                    writer = csv.DictWriter(file, set.union(*[set(r.keys()) for r in records]))
+                f_obj = open(args.path, 'w')
```
Comment (xuehui1991) [inline tools/nni_cmd/nnictl_utils.py]: suggest using "with open(...) as file" to make sure you have the same style with line 279.

Comment (QuanluZhang) [reply tools/nni_cmd/nnictl_utils.py]: good suggestion.

File: tools/nni_cmd/nnictl_utils.py [hunk 2]
```diff
@@ -482,25 +461,25 @@ def export_trials_data(args):
...
+                f_obj.write(response.text)
+                f_obj.close()
+            else:
```
Comment (leckie-chn) [inline tools/nni_cmd/nnictl_utils.py]: suggest using
```python
elif args.type=='csv':
...
else:
    raise ValueError("export file type not supported, please use json or csv instead")
```
 here

Comment (QuanluZhang) [reply tools/nni_cmd/nnictl_utils.py]: good

Comment (SparkSnail) [reply tools/nni_cmd/nnictl_utils.py]: Do not suggest to use raise ValueError(''), it is not friendly for users in command line terminal. Use print_error('') instead.

Comment (QuanluZhang) [reply tools/nni_cmd/nnictl_utils.py]: @SparkSnail thanks!

Comment (SparkSnail) [reply tools/nni_cmd/nnictl_utils.py]: > suggest using
> 
> ```python
> elif args.type=='csv':
> ...
> else:
>     raise ValueError("export file type not supported, please use json or csv instead")
> ```
> 
> here

Do not suggest to use `raise ValueError('')`, it is not friendly for users in command line terminal. Use `print_error('')` and `exit(1)` instead.

**Pull request merged**